overlord/snapstate/snapstate.go

```diff
@@ -1168,7 +1169,8 @@ func InstallPathMany(ctx context.Context, st *state.State, sideInfos []*snap.Sid
 
 // InstallMany installs everything from the given list of names.
 // Note that the state must be locked by the caller.
-func InstallMany(st *state.State, names []string, userID int) ([]string, []*state.TaskSet, error) {
+// TODO implement support for Flags.Transaction
+func InstallMany(st *state.State, names []string, userID int, flags *Flags) ([]string, []*state.TaskSet, error) {
 	// need to have a model set before trying to talk the store
```

Comment:
this probably needs a TODO about supporting flags == nil as it's ultimately needed for consistency